src/diamond/handler/cloudwatch.py

```diff
@@ -80,7 +83,10 @@ def __init__(self, config=None):
         self.log.debug("Setting InstanceId: " + self.instance_id)
 
         self.valid_config = ('region', 'collector', 'metric', 'namespace',
-                             'name', 'unit')
+                             'name', 'unit', 'autoscaling')
+
+        if self.config['autoscaling']:
+            self.autoscaling = True
```

Comment:
Why assign this to `self.autoscaling` instead of just pulling it from the config when needed?